src/store/index.js

```diff
@@ -229,6 +232,45 @@ export default new Vuex.Store({
         })
       })
     },
+    async getServerInfo() {
```

Comment:
This method throws an exception when trying to login to latest.speckle.dev so I can no longer log in there. I know you had to adjust some CORS settings so the same thing may apply here, or we could just try catch and assume it isn't FE2 in the case of an exception.

Also, I would like to know what the CORS settings were that changed. We need to make sure that it is working for production as well as development.

Reply:
I've fixed your issue now, can you test it again?

We do not have to setup CORS for fe1 because there is no `x-speckle-frontend-2` header on fe1 already. We just need to try catch it elegantly.

Regardless, all connectors should be tested on production.